docs/reference/cluster/stats.asciidoc

```diff
@@ -1307,6 +1307,142 @@ Each repository type may also include other statistics about the repositories of
 
 ====
 
+`ccs`::
+(object) Contains information relating to <<modules-cross-cluster-search, cross-cluster search>> settings and activity in the cluster.
```

Comment:
Note: I usually also ask the PM to review the end user docs.
cc: @naj-h

Reply:
Yes, that's a good idea.

Reply:
Will the include_remotes option be available to enable these metrics? If so, should we document it?

Reply:
@naj-h Yes on both, in the next pull following this one.